connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java

```diff
@@ -138,11 +149,13 @@ private void pause() {
 
                 case STARTED:
                     connector.stop();
+                    metrics.recordPaused();
```

Comment:
this falls through anyway, so this is just repeated

Reply:
ack